addons/purchase/i18n/vi.po

```diff
@@ -2006,7 +2006,7 @@ msgstr "Đơn Mua"
 #. module: purchase
 #: model_terms:ir.ui.view,arch_db:purchase.report_purchaseorder_document
 msgid "Purchase Order #"
-msgstr "Đơn Mua Số"
+msgstr "Đơn Mua Số #"
```

Comment:
```suggestion
msgstr "Đơn Mua Số"
```

Reply:
![image](https://user-images.githubusercontent.com/96491226/187809925-2763f05d-3d2e-4c70-b626-18a6f7de7d1f.png)
---
Anh ơi em thấy trong video cuộc họp hôm 24/8 có note chỗ này mọi người thống nhất là viết cách ra ở hóa đơn sau khi đã in ấy ạ.